spec/features/user_creates_an_order_spec.rb

```diff
@@ -0,0 +1,47 @@
+require "rails_helper"
+
+feature "Existing user places an order" do
+  context "while logged and with a cart with products" do
+    before do
+      user = User.create(first_name: "Jane",
+                         last_name:  "Doe",
+                         email:      "jane@gmail.com",
+                         password:   "password")
+
+      product_1 = Product.create(name:        "Plant1",
+                                 description: "Plant 1 description",
+                                 price:       9.99)
+
+      product_2 = Product.create(name:        "Plant2",
+                                 description: "Plant 2 description",
+                                 price:       19.99)
+
+      user_cart = Cart.new(nil)
+      user_cart.add_item(product_1)
+      user_cart.add_item(product_2)
+
+      allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(user)
+      allow_any_instance_of(ApplicationController).to receive(:cart).and_return(user_cart)
```

Comment:
We think you should be able to break the line before each . so it could look something like:

`allow_any_instance_of(ApplicationController)`
`.to receive(:cart)`
`.and_return(user_cart)`